Firebase/InstanceID/FIRInstanceIDAuthKeyChain.h

```diff
@@ -74,22 +74,19 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  *  Set the data for a given service and account with a specific accessibility. If
- *  accessibility is NULL we use `kSecAttrAccessibleAlwaysThisDeviceOnly` which
+ *  accessibility is NULL we use `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly` which
```

Comment:
Looks like the comment should be updated after removing `accessibility` parameter.

Reply:
good catch!